### ISSUE TITLE
Tardis not loading.

### ISSUE THREAD

**Issue by DilanRona** [eccentricdevotion/TARDIS#110]
```
[09:10:43] [Server thread/INFO]: Kento_Jackson issued server command: /tardis
[09:10:43] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Cannot execute command 'tardis' in plugin TARDIS v4.1.0-b1862 - plugin is disabled.
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:37) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.dispatchCommand(CraftServer.java:710) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.PlayerConnection.handleCommand(PlayerConnection.java:1670) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1510) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:47) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:13) ~[server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.TickTask.run(SourceFile:18) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeTask(SourceFile:135) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.executeNext(SourceFile:114) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.MinecraftServer.aW(MinecraftServer.java:898) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.MinecraftServer.executeNext(MinecraftServer.java:891) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.IAsyncTaskHandler.awaitTasks(SourceFile:123) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.MinecraftServer.sleepForTick(MinecraftServer.java:875) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:808) [server.jar:git-Spigot-d05d3c1-a2275f1]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

Latest Tardis, and TARDISChunkGenerator is installed.
worldedit-bukkit-7.0.1-SNAPSHOT-dist.jar build 4463
worldguard-bukkit-7.0.1-SNAPSHOT-dist.jar build 1936

**Comment by eccentricdevotion** [comment]
Most probably wrong version of TARDISChunkGenerator installed.
Need to see the entire log showing server startup...

**Comment by DilanRona** [comment]
Complete server log
https://pastebin.com/dbdYGLU4

**Comment by eccentricdevotion** [comment]
`[TARDIS]This plugin requires ProtocolLib to be v4.5.0 or higher, disabling...`

**Comment by DilanRona** [comment]
https://www.spigotmc.org/resources/protocollib.1997/

It shows ProtocolLib 4.4.0. Unless you are using the latest dev build that is. Are you?

**Comment by eccentricdevotion** [comment]
As it says on the page you linked to...
------
ProtocolLib
If you're running 1.14, use the latest dev build.
------

**Comment by DilanRona** [comment]
Updated to the latest dev build, 4.5.0. Still not working.

https://pastebin.com/Wvjm3w6p

**Comment by eccentricdevotion** [comment]
Looks like you have an invalid recharger entry in your config. Maybe a world that no longer exists on the server?

**Comment by DilanRona** [comment]
rechargers:
  Imperium:
    world: Oumuamua
    x: -139
    y: 67
    z: 151

**Comment by DilanRona** [comment]
That section is properly spaced in my config. Do I need to remove it from the config, and recreate the recharger in the server again?

**Comment by eccentricdevotion** [comment]
No found the problem, try build 1864 from the Jenkins server.

**Comment by DilanRona** [comment]
Ok, its working again. Thank you

**Comment by eccentricdevotion** [comment]
Fixed by 277b63e
